services/core-api/tests/services/test_document_generation_service.py

```diff
@@ -4,18 +4,17 @@
 from app.api.services.document_generator_service import DocumentGeneratorService
 from app.api.document_generation.models.document_template import DocumentTemplate
 
+              # def test_missing_template_returns_fail(test_client, db_session, auth_headers):
```

Comment:
Guessing this is being worked on?

Reply:
ooops. that test fails locally, but should be on for the pipeline.